frontend/plugins/tourism_ui/src/modules/pms/components/pmsFormFields/general/Discount.tsx

```
@@ -0,0 +1,76 @@
+    name: 'discounts',
+  });
+
+  return (
+    <div className="space-y-4">
+      <Button onClick={() => append({})}>
+        <IconPlus />
+        Add discount
+      </Button>
+
+      {fields.map((field, index) => (
+        <div className="flex gap-6 items-end">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)